app/src/main/java/com/kickstarter/viewmodels/DeepLinkViewModel.java

```diff
@@ -33,11 +40,16 @@
 
     /** Emits when we should start the {@link com.kickstarter.ui.activities.ProjectActivity} with pledge sheet expanded. */
     Observable<Uri> startProjectActivityForCheckout();
+
+    /** Emits when we should finish the current activity */
+    Observable<Void> finisDeeplinkActivity();
```

Comment:
nit: typo in method name, missing an "h", should be `finishDeeplinkActivity`

Reply:
Fixed!